gateway/assets/dag.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8" />
+  <meta name="description" content="A DAG hosted on IPFS: {{ .Path }}.">
```

Comment:
`IPFS: /ipfs` will look redundant, maybe:

```suggestion
  <meta name="description" content="Content-addressed {{.CodecName}} document hosted on IPFS.">
```